README.md

```diff
@@ -32,6 +32,25 @@ console.log(one) // [1]
 
 function swap(array, first_index, second_index) {
   // -- YOUR ANSWER HERE --
+   if (array === []) {
+   return array;
+ }
+ if (first_index < 0) {
+   return array;
+ }
+ if (second_index < 0) {
+   return array;
+ }
```

Comment:
What if the indexes are greater than the length of the array?
Also, you could merge these three conditionals into one.